notification-service/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import 'dotenv/config';
+import { sendMail, setupMailer } from './mailer';
+import { setupWebSocket, subscribe } from './websocket';
+import { isNetworkSupported } from './constants/NETWORKS';
+import { ETHEREUM_NETWORK } from './variables';
+import { setupReceiverList } from './recievers';
+
+const start = async function () {
+    isNetworkSupported(ETHEREUM_NETWORK);
+    setupReceiverList();
+    const transporter = await setupMailer();
+    const wsProvider = setupWebSocket();
+
+    subscribe(wsProvider, mailData => sendMail(transporter, ETHEREUM_NETWORK, mailData));
```

Comment:
Why does subscriber already emits `mailData`? I though it would emit event + maybe a subscription id, which then will be transformed into email inside `sendMail` or some other third `generateEmail` function?